core/src/main/java/org/apache/iceberg/encryption/AesGcmOutputStream.java

```diff
@@ -95,6 +102,10 @@ public void write(byte[] b, int off, int len) throws IOException {
 
   @Override
   public long getPos() throws IOException {
```

Comment:
Iceberg manifest writer calls getPos after closing the avro stream. Wrong value was returned.